appcelerator/titanium_mobile
# [TIMOB-25540] Android: Fix sizeOf() calculation to prevent IllegalStateException

- `onTrimMemory()` can cause an exception when attempting to release memory
- Fix `sizeOf()` calculation to prevent `IllegalStateException`

###### TEST CASE
- Download an image first
- Run app and select image from Downloads folder
- Press back to exit app, app should not crash
```JS
Ti.Media.openPhotoGallery({
  success:function(event) {
    var image = event.media.imageAsResized(300, 300);
    console.log('resized image');
  }
});
```

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-25540)
